### PR TITLE
Add Russian localization

### PR DESCRIPTION
**Pull request by unchase**
Also fix some small bugs

### CODE REVIEW

**Comment by unchase** [comment]
@iammukeshm ready for review

**Comment by iammukeshm** [comment]
Hi @unchase , what are the other bug fixes that you had mentioned?

File: BlazorHero.CleanArchitecture/Client/Pages/Identity/Profile.razor.cs [hunk 1]
```diff
@@ -84,7 +84,7 @@ private async Task UploadFiles(InputFileChangeEventArgs e)
                 {
                     foreach (var error in result.Messages)
                     {
-                        _snackBar.Add(error, Severity.Success);
+                        _snackBar.Add(localizer[error], Severity.Error);
```
Comment (unchase) [inline BlazorHero.CleanArchitecture/Client/Pages/Identity/Profile.razor.cs:87]: There is a small bug

File: BlazorHero.CleanArchitecture/Client/Pages/Identity/Profile.razor.cs [hunk 2]
```diff
@@ -110,7 +110,7 @@ private async Task DeleteAsync()
                 {
                     foreach (var error in data.Messages)
                     {
-                        _snackBar.Add(error, Severity.Success);
+                        _snackBar.Add(localizer[error], Severity.Error);
```
Comment (unchase) [inline BlazorHero.CleanArchitecture/Client/Pages/Identity/Profile.razor.cs:113]: There is a small bug

**Comment by iammukeshm** [comment]
Cool! Thanks, a lot. Any suggestions/feedback regarding this project?

**Comment by unchase** [comment]
> Cool! Thanks, a lot. Any suggestions/feedback regarding this project?

I will try to collect data for feedback in the coming days or add it in the form of separate PRs.

**Pull request merged**